content/docs/guides/contribute/review-process.en.md

```diff
@@ -11,10 +11,10 @@ sequenceDiagram
   R->>A: Answers yes or no
   
   loop Loop between author and reviewer
-    R-->>+A: Comments, asks for changes
+    R-->>A: Comments, asks for changes
```

Comment:
I just don't really know what this fix is for

Reply:
I believe it's responsible for this box 
![image](https://github.com/OpenRailAssociation/osrd-website/assets/142150521/8e364c42-b9b4-4c96-be95-3553a4c31c94)

I removed the box in order to get consistency across languages (and I don't think the box is necessary), but I could change the other way around if needed.